internal/integrationtests/scripts/e2e_integration_test.sh

```diff
@@ -5,6 +5,22 @@ set -eu
 export DIVIDER="----------------------------------------"
 # prepare
 
+wait_for_server() {
+  local endpoint=$1
+  local max_wait_time=$2
+
+  SECONDS=0
```

Comment:
❓ Is `SECONDS`  a special variable that auto-increments ?

Reply:
Yes, it's a special built-in bash variable.

the [docs](https://man7.org/linux/man-pages/man1/bash.1.html) say
> **SECONDS**
>
> Each time this parameter is referenced, it expands to the
>  number of seconds since shell invocation.  If a value is
>  assigned to SECONDS, the value returned upon subsequent
>  references is the number of seconds since the assignment
>  plus the value assigned.  The number of seconds at shell
>  invocation and the current time are always determined by
>  querying the system clock.  If SECONDS is unset, it loses
>  its special properties, even if it is subsequently reset.